projects/auth0-angular/src/lib/abstract-navigator.ts

```diff
@@ -21,9 +21,7 @@ export class AbstractNavigator {
    */
   navigateByUrl(url: string): void {
     if (this.router) {
-      setTimeout(() => {
-        this.router.navigateByUrl(url);
-      }, 0);
+      this.router.navigateByUrl(url);
```

Comment:
@stevehobbsdev would you happen to remember what situations we should be verifying in order to guarantee this change doesn't break anything?

Reply:
I think it was any time that the router was called to redirect away, like after `handleRedirectCallback`. I remember noticing at the time that navigation just didn't work until I put this hack in.

I probably got that hack from StackOverflow or something; I should have been more diligent about including a link.

I guess if the router nav works after handling the callback just fine then this could be removed.

Reply:
Okay, this worked fine. Let me test it a few times more and then I will remove the draft status.

Reply:
Ye I gave it a few more tests and I do not see anything behaving unexpectedly